File: codes/php/object_values.php
<?php

echo("\n// JavaScript-like Object.values() in PHP associative-array" . "\n");

$friend = [
    "name" => "Alisa",
    "country" => "Finland",
    "age" => 25
];
echo("friend: " . str_replace("/\n$/", "", json_encode($friend, JSON_PRETTY_PRINT)) . "\n");

echo("friend values: " . str_replace(",", ", ", str_replace(":", ": ", json_encode(array_values($friend)))) . "\n");
// friend values: ["Alisa", "Finland", 25]

File: codes/dart/object_values.dart
import "dart:convert";

void main() {
    dynamic jsonStringify(dynamic anything, { bool pretty = false, String indent = "    " }) {
		if (pretty == true) {
			final jsonEncoder = new JsonEncoder.withIndent(indent);
			return jsonEncoder.convert(anything);
		}
		final jsonEncoder = new JsonEncoder();
		return jsonEncoder.convert(anything).split(',').join(', ');
    }

	print("\n// JavaScript-like Object.values() in Dart Map");

    final friend = <String, dynamic>{
        "name": "Alisa",
        "country": "Finland",
        "age": 25,
    };
    print("friend: ${jsonStringify(friend, pretty: true)}");

	print("friend values: ${jsonStringify(friend.values.toList())}");
	// friend values: ["Alisa", "Finland", 25]
}
